eraser-map.yaml

```diff
@@ -832,10 +831,10 @@ layers:
         data: { source: route_icon }
         draw:
             icons:
-                priority: 1
                 sprite: route-start
                 size: [38px,49px]
                 collide: false
+                anchor: top
```

Comment:
Talking more with Chuck about this, we should be using a different artwork asset for this (more like a arrow in a circle badge), and center (default anchor) would make more sense for this.